### PR TITLE
Feature/17

### LINKED ISSUE #17: Twig HTML helper issue
I've faced with some trouble. When i tried to use the next code

```
{% set favoriteButtonOptions = {'class': 'btn btn-default spec''} %}
{{ html.addCssClass(favoriteButtonOptions, 'btn-primary') }}
```

i got an error https://www.dropbox.com/s/55h033tiy9i19ec/Screenshot%202015-04-01%2011.43.45.png?dl=0

Seems like variable passed not as reference.

### PR DESCRIPTION
**Pull request by amarox**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | fix #17

### CODE REVIEW

**Comment by amarox** [comment]
draft

File: html/CssStyle_TokenParser.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace yii\twig\html;
+
+class CssStyle_TokenParser extends BaseCss_TokenParser
+{
+    public function getNodeClass()
+    {
+        return StyleClassNode::class;
```
Comment (samdark) [inline html/CssStyle_TokenParser.php]: Current twig version is compatible with PHP 5.4.

**Comment by samdark** [comment]
Should be covered by unit tests.

File: docs/guide-ru/html-helper.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+Html хелпер
```
Comment (samdark) [inline docs/guide-ru/html-helper.md]: English docs should be updated. Are you OK doing it? If not, I can take this part.

Comment (amarox) [inline docs/guide-ru/html-helper.md]: Yes, i can do it. Thaks! I planned it.

**Pull request merged**